WordPress/Classes/ViewRelated/NUX/Signin2FAViewController.swift

```diff
@@ -75,6 +76,17 @@ import WordPressShared
     // MARK: Configuration Methods
 
 
+    /// Assigns localized strings to various UIControl defined in the storyboard.
+    ///
+    func localizeControls() {
+        verificationCodeField.placeholder = NSLocalizedString("Verification code", comment: "two factor code placeholder")
+
+        let submitButtonTitle = NSLocalizedString("VERIFY", comment: "Title of a button. The text should be uppercase.").localizedUppercaseString
+        submitButton.setTitle(submitButtonTitle, forState: .Normal)
+        submitButton.setTitle(submitButtonTitle, forState: .Highlighted)
```

Comment:
I see, `UIControlState` is mapped to an `OptionSetType` because it's a `NS_OPTIONS`. That doesn't necessarily means that `setTitle(_:forState:)` will handle all the passed options :(

I would have expected that to work, maybe in a future UIKit version